apps/keyboard/js/layouts/vi-Qwerty.js

```diff
@@ -7,6 +7,7 @@ Keyboards['vi-Qwerty'] = {
   shortLabel: 'Vi',
   menuLabel: 'Tiếng Việt (QWERTY)',
   imEngine: 'vietnamese',
+  basicLayoutKey: 'AĂÂ',
```

Comment:
I don't think this is correct. @tedders1 what's the label for Vietnamese, or can we keep it as default (`ABC`)?